lib-rca/src/main/java/com/facebook/openwifi/librca/inputs/Configs.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.inputs;
+
+/** Define root cause analysis configuration parameters */
+public final class Configs {
+
+	/** Default look-back window in minutes */
+	public static final int DEFAULT_DETECTION_WINDOW = 360;
+
+	// Default KPI calculation parameters
+	/** Default minimum acceptable estimated throughput (Mbps) */
+	public static final double DEFAULT_MIN_ESTIMATED_THROUGHPUT = 10;
+	/**
+	 * Default percentile (units are %) of estimated throughputs to use as the
+	 * KPI
+	 */
+	public static final double DEFAULT_THROUGHPUT_AGGREGATION_PERCENTILE = 10.0;
+	/** Default maximum acceptable latency (ms) */
+	public static final int DFEAULT_MAX_LATENCY_THRESHOLD = 50;
+	/** Default maximum acceptable jitter (ms) */
+	public static final int DEFAULT_MAX_JITTER_THRESHOLD = 20;
+	/**
+	 * Default maximum acceptable disconnection rate (disconnetions per hour)
+	 */
+	public static final int DEFAULT_MAX_DISCONNECTION_RATE = 20;
+
+	// Default high Level metrics parameters
+	/** Default minimum acceptable tx rate (Mbps) */
+	public static final double DEFAULT_MIN_TX_RATE = 50;
+	/** Default maximum acceptable Packet Error Rate (PER) (units are %) */
+	public static final double DEFAULT_MAX_PER = 10.0;
+	/** Default minimum acceptable idle airtime (units are %) */
+	public static final double DEFAULT_MIN_IDLE_AIRTIME = 10.0;
+	/** Default maximum acceptable number of clients for one radio */
+	public static final int DEFAULT_MAX_NUM_CLIENTS = 10;
+
+	// Default low Level metrics parameters
+	/** Default minimum acceptable RSSI (dBm) */
+	public static final int DEFAULT_MIN_RSSI = -70;
+	/** Default maximum acceptable noise (dBm) */
+	public static final int DEFAULT_MAX_NOISE = -95;
+	/** Default maximum acceptable intf airtime (units are %) */
+	public static final double DEFAULT_MAX_INTF_AIRTIME = 75.0;
+	/** Default maximum acceptable number of neighbors */
+	public static final int DEFAULT_MAX_NUM_NEIGHBORS = 10;
+	/** Default minimum acceptable client bandwidth (MHz) for non-2G bands / */
+	public static final int DEFAULT_MIN_CLIENT_BANDWIDTH = 80;
+	/** Default minimum acceptable AP bandwidth (MHz) for non-2G bands */
+	public static final int DEFAULT_MIN_AP_BANDWIDTH = 80;
+	/** Default minimum acceptable self airtime ratio (units are %) */
+	public static final double DEFAULT_MIN_SELF_AIRTIME_RATIO = 25.0;
+	/** Default maximum acceptable tx dropped ratio (units are %) */
+	public static final double DEFAULT_MAX_TX_DROPPED_RATIO = 0.1;
+
+
+	/** Look-back window in minutes */
+	public final int detectionWindow;
```

Comment:
all of these variable names should include the units (as a suffix), particularly time units